app/src/main/java/com/iet/ietians_diary/DashboardFragment.java

```diff
@@ -115,19 +120,44 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         DashboardAllFeaturesRecyclerViewAdapter dashboardAllFeatures = new DashboardAllFeaturesRecyclerViewAdapter(list, getContext(),this  );
         allFeatures.setAdapter(dashboardAllFeatures);
 
-
         return rootView;
     }
 
     @Override
     public void clickListener(int position) {
-        String s= String.valueOf(position);
-        Intent i = new Intent(getContext(), SelectDetails.class);
-        //Toast.makeText(getContext(), s,  Toast.LENGTH_SHORT).show();
-        //Bundle extras = getActivity().getIntent().getExtras();
-        //String name = extras.getString("title");
-        i.putExtra("position", s);
-        startActivity(i);
-
+        AlertDialog.Builder dialogBuilder = new AlertDialog.Builder(getContext());
```

Comment:
This is okay for now, but we'll replace it with MaterialAlertiDialogBuilder in future.